python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -149,34 +139,46 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                   .join(", "),
             },
             {
-              label: "Pending",
+              label: "Number of pending tasks",
               value: actor.taskQueueLength.toLocaleString(),
+              tooltip:
+                "The number of tasks that are currently pending to execute on this actor. If this number " +
+                "remains consistently high, it can indicate that this actor is a bottle-neck in your program.",
             },
             {
-              label: "Executed",
+              label: "Number of executed tasks",
               value: actor.numExecutedTasks.toLocaleString(),
+              tooltip:
+                "The number of tasks this actor has executed throughout its lifetimes.",
             },
             {
-              label: "NumObjectIdsInScope",
+              label: "Number of object IDs in scope",
               value: actor.numObjectIdsInScope.toLocaleString(),
+              tooltip:
+                "The number of objects ids that this actor is keeping in scope via its internal state. " +
+                "This does not imply that the objects are in active use or colocated on the node with the actor " +
+                "currently. This can be useful for debugging memory leaks.",
             },
             {
-              label: "NumLocalObjects",
+              label: "Number of local objects",
               value: actor.numLocalObjects.toLocaleString(),
+              tooltip:
+                "The number of objects that this actor has stored in memory on the node. This can be useful for " +
```

Comment:
```suggestion
                "The number of small objects that this actor has stored in its local in-process memory store. This can be useful for " +
```